public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -18,6 +19,7 @@ import React from 'react';
 import { LokiQuery } from '../../../plugins/datasource/loki/types';
 import { getFieldLinksForExplore } from '../utils/links';
 
+const splunkUID = 'PD90232BAD06BE469';
```

Comment:
We can't rely on this hard-coded uid, so we'll need another way to identify what type of datasource it is. On line 80 (`const dataSourceSettings = getDatasourceSrv().getInstanceSettings(traceToLogsOptions.datasourceUid);)`, dataSourceSettings has a type field which we can use to tell if it's Loki or Splunk